backbone.js

```diff
@@ -518,9 +518,9 @@
       return _.clone(this._previousAttributes);
     },
 
-    // Fetch the model from the server. If the server's representation of the
-    // model differs from its current attributes, they will be overridden,
-    // triggering a `"change"` event.
+    // Fetch the model from the server. If the models differ the two are merged. 
```

Comment:
Probably don't even need the "if the models differ" part. It's merged from the server response regardless.